pages/board.js

```diff
@@ -2,12 +2,38 @@ import { useState } from "react";
 import TaskCard from "../components/TaskCard/TaskCard";
 import styled from "styled-components";
 
-export default function Board({ tasks }) {
+export default function Board({ tasks, updateTaskParent }) {
+  const updateTask = function (task) {
+    updateTaskParent(task);
+  };
+
+  const uniqueCategories = tasks.reduce((categories, task) => {
+    if (!categories.includes(task.category)) {
+      categories.push(task.category);
+    }
+    return categories;
+  }, []);
+
+  const TaskList = ({ tasks }) => {
+    return (
+      <>
+        {tasks.map((task) => (
```

Comment:
Mapping over an array should result in a list element with list items to be accessible.